<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height> <!-- 调整窗口高度 -->
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件压缩与解压缩</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 标题标签 -->
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>文件压缩与解压缩工具</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20pt; font-weight: bold; color: #2E86C1;</string>
    </property>
   </widget>

   <!-- 压缩按钮 -->
   <widget class="QPushButton" name="compressButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件并压缩</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12pt; background-color: #28B463; color: white; border-radius: 10px;</string>
    </property>
   </widget>

   <!-- 解压按钮 -->
   <widget class="QPushButton" name="decompressButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件并解压</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12pt; background-color: #C0392B; color: white; border-radius: 10px;</string>
    </property>
   </widget>

   <!-- 压缩状态标签 -->
   <widget class="QLabel" name="compressStatusLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>压缩状态：等待操作</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10pt; color: #34495E;</string>
    </property>
   </widget>

   <!-- 解压状态标签 -->
   <widget class="QLabel" name="decompressStatusLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>解压状态：等待操作</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10pt; color: #34495E;</string>
    </property>
   </widget>

   <!-- 压缩率标签 -->
   <widget class="QLabel" name="compressionRatioLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>压缩率：N/A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10pt; color: #1F618D;</string>
    </property>
   </widget>

   <!-- 日志标签 -->
   <widget class="QLabel" name="logLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>700</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>操作日志：</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12pt; font-weight: bold; color: #5D6D7E;</string>
    </property>
   </widget>

   <!-- 日志文本框 -->
   <widget class="QPlainTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>700</width>
      <height>250</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 10pt; color: #2C3E50; background-color: #F2F4F4; border: 1px solid #BDC3C7; border-radius: 5px;</string>
    </property>
   </widget>

  </widget>

  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
